scripts/angular-version.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+BRANCH=$1
+deprecation_config_file="../deprecation-config.yml"
+# Use yq to extract the version for Angular Core
+angular_version=$(yq eval '.npm["angular/core"][0].version' "$deprecation_config_file")
+echo "Current version is: ${angular_version}"
+
+date_to_timestamp() {
+    # If debugging locally use below date instead   
+    # date -jf "%Y-%m-%d" "$1" +%s
+    date -d "$1" +%s
+}
+
+angular_eol_data=$(curl -s https://endoflife.date/api/angular.json | jq -c '.[]')
+
+# Get the current date in Unix timestamp
+current_date=$(date +%s)
+current_date="1727609985"
```

Comment:
Set this to some point in the future for now to test the workflow, will raise and revert this line after